.github/workflows/pythonpackage.yml

```diff
@@ -22,28 +22,27 @@ jobs:
           - 6379:6379
     steps:
       - uses: actions/checkout@v1
```

Comment:
Optional: Might be worth updating this, I'm fairly certain it'll run into issues with the node and/or output deprecation.